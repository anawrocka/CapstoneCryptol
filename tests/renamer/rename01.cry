module rename01 where

// Rename a top-level function with one bound parameter
f x = x
