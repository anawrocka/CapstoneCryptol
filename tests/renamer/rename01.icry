:l rename01.cry
