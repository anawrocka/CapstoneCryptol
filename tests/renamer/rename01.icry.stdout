Loading module Cryptol
Loading module Cryptol
Loading module rename01
